### PR TITLE
docs(lunch_queue) correct grammar

### PR DESCRIPTION
**Pull request by davhojt**
Language review of a single exercise lunch_queue

It has been done alone, as it was not done as part of a previous language review. It is now only in exam >3

### CODE REVIEW

File: subjects/lunch_queue/README.md [hunk 1]
```diff
@@ -2,27 +2,19 @@
...
-- `search`, which returns a tuple with the information of a given person `id`
+- `new`: which will initialize the `Queue`.
+- `add`: which adds a person to the queue.
+- `invert_queue`: which reverses the queue.
+- `rm`: which removes the person who finished ordering their food. The removal should respect the FIFO method (first in first out). It should return the person's details.
+- `search`: which returns the details for a given person's `name`.
 
-You must also create a type called `Link`. This will be the connection of the structures `Queue` and `Person`.
-Do not forget that this will be a recursion type and it must point to `None` if there is no persons.
+You must also create a type named `Link`. This will be the connection of the structures `Queue` and `Person`. This will be a recursion type and it must point to `None` when the person is the last person in the queue.
```
Comment (MSilva95) [inline subjects/lunch_queue/README.md:15]: Is it in the last person or when there are no more people?

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Just need to confirm if the line 15 is correct or not

File: subjects/lunch_queue/README.md (hunk 1 above)
Comment (davhojt) [reply subjects/lunch_queue/README.md:15]: Ah, I think you're right to point this out, as I guess we are talking about the `Queue` here, not `Person`.
The `Link` in `Queue` should point to a `Person` or `None` if there are no people.
The `Link` in `Persons` which we don't actually describe here should point to `None` when it is at the end of the *linked list* of people.

Comment (davhojt) [reply subjects/lunch_queue/README.md]: I changed it here to make it a little more generic for `Queue` *and* `Person`: https://github.com/01-edu/public/pull/1109/commits/84226861cfa1f2116644ccf5479f68da1b6e560c

Comment (MSilva95) [reply subjects/lunch_queue/README.md]: Cool

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**